src/nvim/sha256.c

```diff
@@ -8,8 +8,6 @@
 /// at your choice.
 ///
 /// Vim specific notes:
-/// Functions exported by this file:
-///  2. sha2_seed() generates a random header.
 /// sha256_self_test() is implicitly called once.
 
 #include <stddef.h>        // for size_t
```

Comment:
> I didn't check that

It's fine, it's just that this commit is tiny so we might as well add some minor related cleanups.

Anyways, there's no need to resubmit: just use `git push --force`.